default/stringtables/en.txt

```diff
@@ -11984,8 +11984,8 @@ Confederation
 PLC_CONFEDERATION_DESC
 '''Planets of the empire are organized in a confederacy of independent systems.
 
-• Increases [[metertype METER_TARGET_INFLUENCE]] on colonies disconnected from the capital by [[value PLC_CONFEDERATION_SUPPLY_DISCONNECTED_INFLUENCE_FLAT]]
-• Increases [[metertype METER_TARGET_HAPPINESS]] on colonies disconnected from the capital and regional administrative centres by [[value DISCONNECTED_FROM_CAPITAL_AND_REGIONAL_ADMIN_STABILITY_PENALTY]]
+• Removes [[metertype METER_TARGET_INFLUENCE]] penalty on colonies disconnected from the capital
```

Comment:
Could be misleading... It compensates for a specific penalty, but doesn't prevent another source of such a penalty, and doesn't do anything if that penalty wasn't present.

Reply:
Hmm I'll have a think about it and see if I can come up with clearer wording. If stability penalties were in the pedia I could link to the penalty. Maybe its worth creating a pedia entry for stability modifiers? That might be worthwhile in itself.

Reply:
It was possibly not intended, but Sly and Laenfa are a lot happier with confederation on disconnected planets. The get the bonus without having the penalty.